crates/cargo-winrt/src/main.rs

```diff
@@ -368,6 +379,27 @@ USAGE:
     }
 }
 
+#[derive(Debug)]
+struct Version {}
```

Comment:
perhaps `struct Version;` is the better syntax?

Reply:
Sure!